source/actions3/actions/CLSSIS3820ScalerDarkCurrentMeasurementAction.h

```diff
@@ -4,6 +4,8 @@
 #include "actions3/AMAction3.h"
 #include "actions3/actions/CLSSIS3820ScalerDarkCurrentMeasurementActionInfo.h"
 
+#define CLSSIS3820SCALERDARKCURRENTMEASUREMENTACTION_WAIT_SECONDS 0.5
```

Comment:
I assume this flat wait value is long enough to consistently give enough time for the values to populate?